installing/installing_gcp/installing-gcp-user-infra-vpc.adoc

```diff
@@ -5,7 +5,9 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-In {product-title} version {product-version}, you can install a cluster into a shared Virtual Private Cloud (VPC) on Google Cloud Platform (GCP) that uses infrastructure that you provide. In this context, a cluster installed into a shared VPC is a cluster that is configured to use a VPC from a project different from where the cluster is being deployed. For more information, see Google's documentation on link:https://cloud.google.com/vpc/docs/shared-vpc#shared_vpc_networks[shared VPC networks].
+In {product-title} version {product-version}, you can install a cluster into a shared Virtual Private Cloud (VPC) on Google Cloud Platform (GCP) that uses infrastructure that you provide. In this context, a cluster installed into a shared VPC is a cluster that is configured to use a VPC from a project different from where the cluster is being deployed. For more information, see link:https://cloud.google.com/vpc/docs/shared-vpc#shared_vpc_networks[shared VPC networks] in the GCP documentation.
+
```

Comment:
shared VPC networks is a section of Shared VPC overview. So I would suggest to keep one link. I prefer to keep the Shared VPC overview link. We can remove "For more information, see link:https://cloud.google.com/vpc/docs/shared-vpc#shared_vpc_networks[shared VPC networks] in the GCP documentation."